### PR TITLE
HBASE-28906 Run nightly tests with multiple Hadoop 3 versions

### PR DESCRIPTION
**Pull request by stoty**
The nightly tests won't run on this PR.

See https://ci-hbase.apache.org/job/Test%20script%20for%20nightly%20script/job/HBASE-28846/ 
for the hacked-up test PR which tests the contents of this change

### CODE REVIEW

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/1/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/1/console in case of problems.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 46s |  |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --author-ignore-list --blanks-eol-ignore-file --blanks-tabs-ignore-file --quick-hadoopcheck  |
|||| _ Prechecks _ |
|||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  9s |  |  Maven dependency ordering for branch  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  |  Maven dependency ordering for patch  |
|||| _ Other Tests _ |
|  |   |   1m 47s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.47 ServerAPI=1.47 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/1/artifact/yetus-jdk17-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6356 |
| Optional Tests |  |
| uname | Linux 4712553926bf 5.4.0-195-generic #215-Ubuntu SMP Fri Aug 2 18:28:05 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / e3c2a51b73dd81f12085fcc636c8561ce6521b5f |
| Max. process+thread count | 40 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/1/console |
| versions | git=2.34.1 maven=3.9.8 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 34s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  spotless  |   0m 43s |  |  branch has no errors when running spotless:check.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  4s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  shellcheck  |   0m  2s |  |  No new issues.  |
| +1 :green_heart: |  spotless  |   0m 39s |  |  patch has no errors when running spotless:check.  |
|||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  |  ASF License check generated no output?  |
|  |   |   3m 10s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.47 ServerAPI=1.47 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6356 |
| Optional Tests | dupname asflicense codespell detsecrets shellcheck shelldocs spotless |
| uname | Linux b8548ab04815 5.4.0-192-generic #212-Ubuntu SMP Fri Jul 5 09:47:39 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / e3c2a51b73dd81f12085fcc636c8561ce6521b5f |
| Max. process+thread count | 47 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/1/console |
| versions | git=2.34.1 maven=3.9.8 shellcheck=0.8.0 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/2/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/2/console in case of problems.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 40s |  |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --author-ignore-list --blanks-eol-ignore-file --blanks-tabs-ignore-file --quick-hadoopcheck  |
|||| _ Prechecks _ |
|||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  |  Maven dependency ordering for branch  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  |  Maven dependency ordering for patch  |
|||| _ Other Tests _ |
|  |   |   1m 34s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.47 ServerAPI=1.47 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/2/artifact/yetus-jdk17-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6356 |
| Optional Tests |  |
| uname | Linux 2650b3f0ebd3 5.4.0-192-generic #212-Ubuntu SMP Fri Jul 5 09:47:39 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 832c3abd597ab44c263d72dd4a33aa4dcfe6d73b |
| Max. process+thread count | 41 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/2/console |
| versions | git=2.34.1 maven=3.9.8 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 34s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  spotless  |   0m 45s |  |  branch has no errors when running spotless:check.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  |  No new issues.  |
| +1 :green_heart: |  spotless  |   0m 40s |  |  patch has no errors when running spotless:check.  |
|||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  1s |  |  ASF License check generated no output?  |
|  |   |   3m  0s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6356 |
| Optional Tests | dupname asflicense codespell detsecrets shellcheck shelldocs spotless |
| uname | Linux 3d75d485dfd2 5.4.0-1103-aws #111~18.04.1-Ubuntu SMP Tue May 23 20:04:10 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 832c3abd597ab44c263d72dd4a33aa4dcfe6d73b |
| Max. process+thread count | 46 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/2/console |
| versions | git=2.34.1 maven=3.9.8 shellcheck=0.8.0 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by stoty** [comment]
I agree that this is not very nice, but this was least invasive solution I could think of.

IF we were to stay on Jenkins, then it would be worth either converting all Hadoop tests to a single monster Matrix stage, or the whole Jenkinesfile to a fully scripted pipeline, but hopefully this will only be needed for a few months, and we can spend the effort to convert to GH actions instead.

**Comment by ndimiduk** [comment]
I found the declarative version of these files to be more readable and maintainable than the scripted versions...

Don't hold your breath on a GA implementation anytime soon. Infra hasn't acknowledged my ticket yet.

File: dev-support/hbase-personality.sh [hunk 1]
```diff
@@ -597,8 +618,9 @@ function hadoopcheck_rebuild
     fi
   else
     yetus_info "Setting Hadoop 3 versions to test based on branch-2.6+/master/feature branch rules"
+    # Isn't runnung these tests with the default Hadoop version redundant ?
     if [[ "${QUICK_HADOOPCHECK}" == "true" ]]; then
-      hbase_hadoop3_versions="3.3.6 3.4.0"
```
Comment (Apache9) [inline dev-support/hbase-personality.sh:601]: You can see the pattern, actually we will use the latest one for a minor release line for quick hadoop check...

**Comment by stoty** [comment]
I agree, defining the Hadoop versions both in the personality file and the Jenkinsfile is far from ideal.

We could remove those from the personality, and pass a parameter from the Jenkinsfile, but then we'd have to define them both in the nigthly and precommit jenkinsfiles, so we'd have to maintain two copies either way.

Or we could have some kind of include file that is ready by both Jenkins and Yetus. That way it would in place, but we'd have to add additional logic to parse that in multiple places.

What is your planned release timeline, @Apache9  ? Should we spend some more weeks on this ? The turnaround time for testing these changes is horribly long, basically one run / day.

File: dev-support/hbase-personality.sh [hunk 2]
```diff
@@ -233,8 +246,15 @@ function personality_modules
   # tests respectively.
   if [[ ${testtype} == unit ]]; then
     local tests_arg=""
+
+    if [ -n "${TEST_PROFILE}" ]; then
+      extra="${extra} -P${TEST_PROFILE}"
+    else
+      extra="${extra} -PrunAllTests"
```
[the remark this answers is not in the document]
Comment (stoty) [reply dev-support/hbase-personality.sh:253]: The parameter parsing is already cryptic (for a bash hater like me), adding a default value there would be even more cryptic IMO (and disturb the current relatively easy-to read option processing pattern) .

**Comment by Apache9** [comment]
Have you seen my last review comment? In general this is a workable solution. The only thing is about the hadoop quick check version.

We have already build with 3.3.5 by default, so for hadoop quick check, we'd better test the newest one instead of testing 3.3.5 again...

File: dev-support/hbase-personality.sh (hunk 1 above)
Comment (stoty) [reply dev-support/hbase-personality.sh:601]: Done

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/3/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/3/console in case of problems.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 36s |  |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --author-ignore-list --blanks-eol-ignore-file --blanks-tabs-ignore-file --quick-hadoopcheck  |
|||| _ Prechecks _ |
|||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  |  Maven dependency ordering for branch  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  |  Maven dependency ordering for patch  |
|||| _ Other Tests _ |
|  |   |   1m 32s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.47 ServerAPI=1.47 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/3/artifact/yetus-jdk17-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6356 |
| Optional Tests |  |
| uname | Linux d2eba2b8d299 5.4.0-192-generic #212-Ubuntu SMP Fri Jul 5 09:47:39 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / f42843577296a0736b3f7add5f65823e7bd917c8 |
| Max. process+thread count | 39 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/3/console |
| versions | git=2.34.1 maven=3.9.8 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 16s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  9s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  spotless  |   0m 42s |  |  branch has no errors when running spotless:check.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  |  No new issues.  |
| +1 :green_heart: |  spotless  |   0m 40s |  |  patch has no errors when running spotless:check.  |
|||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  |  ASF License check generated no output?  |
|  |   |   2m 49s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.47 ServerAPI=1.47 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/3/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6356 |
| Optional Tests | dupname asflicense codespell detsecrets shellcheck shelldocs spotless |
| uname | Linux bd8e6b1ce7e6 5.4.0-192-generic #212-Ubuntu SMP Fri Jul 5 09:47:39 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / f42843577296a0736b3f7add5f65823e7bd917c8 |
| Max. process+thread count | 47 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/3/console |
| versions | git=2.34.1 maven=3.9.8 shellcheck=0.8.0 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/4/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/4/console in case of problems.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 38s |  |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --author-ignore-list --blanks-eol-ignore-file --blanks-tabs-ignore-file --quick-hadoopcheck  |
|||| _ Prechecks _ |
|||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  |  Maven dependency ordering for branch  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  |  Maven dependency ordering for patch  |
|||| _ Other Tests _ |
|  |   |   1m 32s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.47 ServerAPI=1.47 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/4/artifact/yetus-jdk17-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6356 |
| Optional Tests |  |
| uname | Linux b4fa4a748ae4 5.4.0-192-generic #212-Ubuntu SMP Fri Jul 5 09:47:39 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 245bea4a57a0dd2b6edac778588eb24d4cff6514 |
| Max. process+thread count | 35 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/4/console |
| versions | git=2.34.1 maven=3.9.8 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 16s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  spotless  |   0m 40s |  |  branch has no errors when running spotless:check.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  5s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  |  No new issues.  |
| +1 :green_heart: |  spotless  |   0m 39s |  |  patch has no errors when running spotless:check.  |
|||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  |  ASF License check generated no output?  |
|  |   |   2m 40s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.47 ServerAPI=1.47 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/4/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/6356 |
| Optional Tests | dupname asflicense codespell detsecrets shellcheck shelldocs spotless |
| uname | Linux 017af88a8f70 5.4.0-192-generic #212-Ubuntu SMP Fri Jul 5 09:47:39 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 245bea4a57a0dd2b6edac778588eb24d4cff6514 |
| Max. process+thread count | 47 (vs. ulimit of 30000) |
| modules | C:  U:  |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-6356/4/console |
| versions | git=2.34.1 maven=3.9.8 shellcheck=0.8.0 |
| Powered by | Apache Yetus 0.15.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by stoty** [comment]
Merged to master.

I'm goint to wait for at least one nightly run before beginning the backports.

**Pull request merged**